src/components/GradesView/GradesView.scss

```diff
@@ -116,9 +116,10 @@ select#ScoreView.form-control {
 [dir=rtl] #course-grade-tooltip .arrow {
     right: initial;
     left: 0;
-    
+
     &:before {
         border-width: 0.4rem 0.4rem 0.4rem 0;
-        border-right-color: $black;
+        border-right-color: black;
+        border-right-color: var(--black);
```

Comment:
Why do we need two values on the `board-right-color`?

Reply:
It is for compatibility with old browsers. @xitij2000 might be able to provide more details as I am not fully aware of it.

Reply:
I think at this point of time the browsers that don't support CSS variables are just IE11 which is not longer supported anyway. So I guess it's OK to just leave the `var` here.

Reply:
@navinkarkera could you just leave line 122 in, and remove line 123? Once you've done that, I can merge this PR.

Reply:
why are we removing the themed black variable?

Reply:
@muselesscreator @schenedx 
>why are we removing the themed black variable?

@xitij2000 has explained why we are removing paragon scss variables [here](https://github.com/openedx/frontend-app-gradebook/pull/258#issuecomment-1243240224)

>could you just leave line 122 in, and remove line 123? Once you've done that, I can merge this PR.

@schenedx We should keep line 123 and remove 122 if I am not wrong.